pkg/build/api/v1/types.go

```diff
@@ -402,6 +402,11 @@ type SourceBuildStrategy struct {
 
 	// ForcePull describes if the builder should pull the images from registry prior to building.
 	ForcePull bool `json:"forcePull,omitempty"`
+
+	// disableImplicitBuild prevents source-to-image from performing a Docker build operation
```

Comment:
yeah, @gabemontero already has a pull open that's updating them all though:
https://github.com/openshift/origin/pull/8477/files#diff-43dd40b4c2b85b388a3d9116729032e4L14